morpholopy/galaxy_data.py

```diff
@@ -98,9 +99,9 @@
 # for brevity, we append individual plots in logical chunks:
 # - spatially resolved and azimuthally averaged KS plots
 for type, label in [
-    ("neutral", "$\\Sigma{}_{{\\rm{}HI}+{\\rm{}H2}}$"),
```

Comment:
These latex expressions look unnecessarily complicated; this is why I simplified this latex code by, among others, using raw strings.